File: src/main/java/net/eduard/api/server/currency/list/CurrencyJHCash.java
package net.eduard.api.server.currency.list;

import net.eduard.api.lib.modules.FakePlayer;
import net.eduard.api.lib.game.ItemBuilder;
import net.eduard.api.lib.storage.annotations.StorageAttributes;
import net.eduard.api.server.currency.SimpleCurrencyHandler;
import org.bukkit.Material;

@StorageAttributes(indentificate = true)
public class CurrencyJHCash extends SimpleCurrencyHandler {
    public CurrencyJHCash() {
        setName("JHCash");
        setDisplayName("Sistema de Cash");
        setSymbol("$");
        setIcon( new ItemBuilder(Material.DIAMOND_BLOCK).name("§aCash"));

    }

    @Override
        public double get(FakePlayer player) {

            return JH_Shop.Main.getAPI().getPontos(player.getName());
        }

        @Override
        public boolean remove(FakePlayer player, double amount) {
            JH_Shop.Main.getAPI().removePontos(player.getName(), amount);
            return true;
        }

        @Override
        public boolean check(FakePlayer player, double amount) {
            return JH_Shop.Main.getAPI().getPontos(player.getName()) >= amount;

        }

        @Override
        public boolean add(FakePlayer player, double amount) {
            JH_Shop.Main.getAPI().addPontos(player.getName(), amount);
            return true;
        }

}

File: src/main/java/net/eduard/api/supports/CurrencyJHCash.kt
package net.eduard.api.supports

import JH_Shop.Main
import net.eduard.api.server.currency.SimpleCurrencySystem
import net.eduard.api.lib.modules.FakePlayer
import net.eduard.api.lib.game.ItemBuilder
import net.eduard.api.lib.storage.annotations.StorageAttributes
import org.bukkit.Material

@StorageAttributes(indentificate = true)
class CurrencyJHCash : SimpleCurrencySystem() {
    override operator fun get(player: FakePlayer): Double {
        return Main.getAPI().getPontos(player.name)
    }

    override fun contains(player: FakePlayer, amount: Double): Boolean {
        return Main.getAPI().getPontos(player.name) >= amount
    }

    override fun remove(player: FakePlayer, amount: Double): Boolean {
        Main.getAPI().removePontos(player.name, amount)
        return true
    }



    override fun add(player: FakePlayer, amount: Double): Boolean {
        Main.getAPI().addPontos(player.name, amount)
        return true
    }

    init {
        name = "JHCash"
        displayName = "Sistema de Cash"
        symbol = "$"
        icon = ItemBuilder(Material.DIAMOND_BLOCK).name("§aCash")
    }
}